### PR TITLE
Reply comment bl

### PR DESCRIPTION
**Pull request by RomanDeveloperAcc**

### CODE REVIEW

File: src/app/component/comments/components/add-comment/add-comment.component.ts [hunk 1]
```diff
@@ -28,7 +34,11 @@ export class AddCommentComponent implements OnInit {
   private isLoggedIn: boolean;
 
   ngOnInit() {
-    this.addElemsToCurrentList();
+    if (this.dataSet.type === 'comment') {
```
Comment (KimAT25) [inline src/app/component/comments/components/add-comment/add-comment.component.ts:37]: Please use ternary operator here

File: src/app/component/comments/services/comments.service.ts [hunk 2]
```diff
@@ -29,6 +29,9 @@ export class CommentsService {
     return this.http.get(`${this.backEnd}econews/comments?ecoNewsId=${this.ecoNewsId}&page=0&size=12`);
   }
 
+  public getAllReplies(id: number): Observable<object> {
+    return this.http.get(`https://greencity.azurewebsites.net/econews/comments/replies/${id}`);
```
Comment (KimAT25) [inline src/app/component/comments/services/comments.service.ts:33]: please add here type and use alias for link

File: src/app/component/comments/services/comments.service.ts [hunk 3]
```diff
@@ -37,6 +40,10 @@ export class CommentsService {
     return this.http.get<number>(`${this.backEnd}econews/comments/count/likes?id=${id}`);
   }
 
+  public getRepliesAmount(id: number): Observable<number> {
+    return this.http.get<number>(`https://greencity.azurewebsites.net/econews/comments/count/replies?parentCommentId=${id}`);
```
Comment (KimAT25) [inline src/app/component/comments/services/comments.service.ts:44]: same as line 33

File: src/app/component/comments/services/comments.service.ts (hunk 2 above)
Comment (RomanDeveloperAcc) [reply src/app/component/comments/services/comments.service.ts]: > please add here type and use alias for link

Fixed

File: src/app/component/comments/services/comments.service.ts (hunk 3 above)
Comment (RomanDeveloperAcc) [reply src/app/component/comments/services/comments.service.ts]: > same as line 33

Fixed

**Pull request merged**